localstack/utils/testutil.py

```diff
@@ -590,7 +590,10 @@ def check_expected_lambda_log_events_length(expected_length, function_name, rege
 
 
 def list_all_log_events(log_group_name: str) -> List[Dict]:
-    logs = aws_stack.connect_to_service("logs")
+    if os.environ.get("TEST_TARGET") == "AWS_CLOUD":
+        logs = boto3.client("logs")
+    else:
+        logs = aws_stack.connect_to_service("logs")
```

Comment:
maybe add the logs client as method parameter and then replace this line with.
`logs = logs or aws_stack.connect_to_service("logs")`

the we don't need the switch